CIScripts/Test/Tests/Protocol/TunnellingWithAgent.Tests.ps1

```diff
@@ -482,7 +482,10 @@ Describe "Tunnelling with Agent tests" {
             Clear-TestConfiguration -Session $Sessions[0] -SystemConfig $SystemConfig
             Clear-TestConfiguration -Session $Sessions[1] -SystemConfig $SystemConfig
         } finally {
-            Merge-Logs -LogSources (New-LogSource -Path (Get-ComputeLogsPath) -Sessions $Sessions)
+            [LogSource[]] $LogSources = New-FileLogSource -Path (Get-ComputeLogsPath) -Sessions $Sessions
+            $LogSources += New-ContainerLogSource -Session $Sessions[0] -Container $Container1ID
```

Comment:
`Session` or `Sessions`? `Container` or `ContainerNames`? Please check parameters' names for New-ContainerLogSource. We know that Powershell can handle parameter name abbreviations, but please use full names as they are declared.